content/applications/hr/employees/new_employee.rst

```diff
@@ -168,18 +184,21 @@ configured skill types and create new skill types. Click :guilabel:`New` and a n
 Type` form appears. Fill out all the details for the new skill type. Repeat this for all the skill
 types needed.
 
-- :guilabel:`Skill Type`: Enter the name of the skill type. This will act as the parent category
-  for more specific skills and should be generic.
-- :guilabel:`Skills`: Click :guilabel:`Add a line` and enter the name for the new skill, then repeat
+- :guilabel:`Skill Type`: enter the name of the skill type. This acts as the parent category for
+  more specific skills and should be generic.
+- :guilabel:`Skills`: click :guilabel:`Add a line` and enter the name for the new skill, then repeat
   for all other needed skills.
-- :guilabel:`Levels`:  Click :guilabel:`Add a line` and a :guilabel:`Create Levels` form appears.
-  Enter the name of the level, and assign a number for the :guilabel:`Progress` percentage (0-100)
-  for that level. Click :guilabel:`Save & New` to save the entry and add another level, or click
-  :guilabel:`Save & Close` to save the level and close the form. Once all the levels are entered,
-  click the :guilabel:`Save manually` icon at the top of the screen. Next, select a default level
-  for this skill type. Click :guilabel:`Set Default` on the line to be the default value. This is
-  typically the lowest level, but any level can be set as the default. A check mark appears in the
-  :guilabel:`Default Level` column, indicating which level is the default.
+- :guilabel:`Levels`: click :guilabel:`Add a line` and enter the name of the level. Next, click into
+  the :guilabel:`Progress` field and enter a percentage (0-100) for that level. Repeat for all
+  additional levels as needed.
+- :guilabel:`Default Level`: click the toggle on the level line to set that level as the default.
+  Typically, the lowest level is set as the default, but any level can be chosen. The toggle turns
+  green, indicating it is the default level for the skill. Only one level can be set as the default.
+
+Once the form is completely filled out, click the :guilabel:`Save manually` icon at the top of the
+screen. Once saved, the :guilabel:`Levels` rearrange in descending order, with the highest level at
+the top, and the lowest at the bottom, regardless of the default level and the order they were
+entered.
 
   .. example::
      To add a math skill set, enter `Math` in the :guilabel:`Name` field. Next, in the
```

Comment:
Thanks for catching that- no save button.no need to save. The save manually is just to rearrange the levels, which is in a tip admonition now. Also, the image is very hard to capture- I tried to make it as large as possible, while still seeing all the information (very hard) so this is as big as I can make it and capture it. I think it's a good example to see how the form looks when filled out, and since I call out all the words/entries I think it's OK. Unless someone has a larger monitor and can capture the image for me, I don't know how I could make it better/sharper. I have this issue with a lot of the larger forms tat take up a big chunk of the screen.